src/preact/base-element.js

```diff
@@ -307,7 +304,7 @@ export class PreactBaseElement extends BaseElement {
     /** @private {?Array} */
     this.contextValues_ = null;
 
-    /** @type {Element | null} */
+    /** @protected {Element | null} */
```

Comment:
is this change intentional? do subclasses utilize `this.container_`?

Reply:
It's used in mutatedAttributesCallback [here](https://github.com/ampproject/amphtml/pull/37309/files#diff-7d1d95d912247f7142f4a510777181aff5dfc1975f5ad4de225cd2dc6b064015R17)

Reply:
carrry on then :)